code/modules/mob/dead/observer/observer.dm

```diff
@@ -538,6 +538,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	if(!target)
 		return 0
+
+	if(istype (target, /mob/living/simple_animal/hostile/megafauna))
```

Comment:
@optimumtact I'm almost certain that's completely irrelevant with the var access upgrades for 511.